### PR TITLE
added TC22

### PR DESCRIPTION
**Pull request by kgold2018**
Added Test Case 22: Add to cart from Recommended items

### CODE REVIEW

File: cypress/cypress/e2e/AE_Tests.cy.js [hunk 1]
```diff
@@ -357,5 +357,35 @@ describe("AE Tests", () => {
     //     expect(text).to.eq('Thank you for your review.');
     // })
   });
-  
+
+
+  it("AE_TestCase22", () => {
+        cy.visit('http://automationexercise.com');
```
Comment (Stavolg) [inline cypress/cypress/e2e/AE_Tests.cy.js:363]: The visit to the main page has been moved to beforeEach.

File: cypress/cypress/e2e/AE_Tests.cy.js [hunk 2]
```diff
@@ -357,5 +357,35 @@ describe("AE Tests", () => {
...
+        cy.scrollTo('bottom')
+
+        cy.get('.recommended_items h2.title')
```
Comment (Stavolg) [inline cypress/cypress/e2e/AE_Tests.cy.js:366]: At the last lesson, we decided to use single-line commands. Earlier, I had already fixed all the commands that were written in multiple lines.

**Review by Stavolg — CHANGES_REQUESTED** [review]
Need Single-Line Commands

**Review by DJKommander — APPROVED** [review]

**Review by afaabb — APPROVED** [review]

**Review by Stavolg — APPROVED** [review]
didn't fix Single-Line Commands

**Pull request merged**